src/AbsRestore.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ProjectRestore;
+
+use Keboola\StorageApi\Client as StorageApi;
+use MicrosoftAzure\Storage\Blob\BlobRestProxy;
+use MicrosoftAzure\Storage\Blob\Models\Blob;
+use MicrosoftAzure\Storage\Blob\Models\ListBlobsOptions;
+use Psr\Log\LoggerInterface;
+
+class AbsRestore extends Restore
+{
+    private BlobRestProxy $absClient;
+
+    private string $container;
+
+    private ?string $blobPrefix = null;
+
+    public function __construct(
+        StorageApi $sapiClient,
+        BlobRestProxy $absClient,
+        string $container,
+        ?LoggerInterface $logger = null
+    ) {
+        $this->absClient = $absClient;
+        preg_match('/([^\/]+)\/?(.+)?/', $container, $match);
+        $this->container = $match[1];
+        if (isset($match[2])) {
+            $this->blobPrefix = (string) $match[2];
```

Comment:
Vysvetlis mi prosim, co je ten `blobPrefix` ... je to to iste co `path` v backup apke, alebo nieco ine?

Reply:
bude lepší na callu